packages/logger/index.d.ts

```diff
@@ -0,0 +1 @@
+export * from './src';
```

Comment:
I believe other packages are exporting type definitions from `./lib`. IIRC, we are not shipping `src` in npm packages.